src/pages/en/toberfests.mdx

```diff
@@ -0,0 +1,96 @@
+
+If you're using github.dev, the Commit menu is in the sidebar below the search magnifying glass. There's no need to push your changes, as they'll be automatically sent to GitHub.
+
+![Screenshot of part of the Visual Studio Code sidebar, showing the Source Control icon](/assets/img/writing/toberfest/toberfest-3.png)
+
+### Branches
+Another key concept of git is branches. At its core, a branch is a path of commits that have at some point been split of the main branch (which you could think of as a trunk) and continued on independently. This is useful in situations where you want to make changes, but don't want to put the files on the main branch at risk of damage, so you can create a branch to keep on working without affecting the main one. In the [quiltmc.org repository](https://github.com/quiltmc/quiltmc.org), for example, anything that is committed to the `stable` branch (which is the main one) is automatically published live on the main website, so we always make sure to develop and test new pages or features on other branches, before sending the changes to the main branch.
+
+When making changes to a fork that you intend to send back to the original repository, it is good etiquette to create a branch for yourself to work on, rather than committing to the main branch. To do this in GitHub Desktop, click the "Current Branch" button, click "New Branch", name it, and click "Create Branch". You will be automatically switched to the new branch.
+![Screenshot showing the branch selection interface of GitHub Desktop](/assets/img/writing/toberfest/toberfest-4.png)
+In github.dev, there's a small button in the bottom-left next to the green GitHub text. Click it, and click "Create New Branch" to create a new branch.
+
+![Screenshot showing the Branch Selection interface of Visual Studio Code](/assets/img/writing/toberfest/toberfest-5.png)
+### Pull Requests
+The last thing you need to know about Git and GitHub is the concept of Pull Requests. Pull requests are GitHub's system for sending changes between branches, or, more importantly, from a fork of a repository back to the original one. To create a Pull Request, visit your fork on GitHub, click current branch in the top left, and select the branch with your changes to switch to it.
+
+![Screenshot highlighting the branch switching interface on github.com](/assets/img/writing/toberfest/toberfest-6.png)
+
+Then click "Contribute" and then click "Create Pull Request". More information about what you need to include in your Pull Requests can be found further down the page.
+
+And that is the basics of how to use git and GitHub. If you're interested in learning more about how git works, there is an excellent visual tutorial available at [https://learngitbranching.js.org](https://learngitbranching.js.org). If you get stuck, don't be afraid to ask for help on [Discord](https://discord.quiltmc.org/toolchain).
```

Comment:
Again direct links should just work:
```diff
- [https://learngitbranching.js.org](https://learngitbranching.js.org)
+ https://learngitbranching.js.org
```
And that we should use the `site` variables for links to our Discord server and stuff like that
```diff
---
# ...
---
+ import site from "@data/Site";

<!-- snip -->

- to ask for help on [Discord](https://discord.quiltmc.org/toolchain).
+ to ask for help on <a href={site.discordCommunity}>Discord</a>.